helm-chart/values.yaml

```diff
@@ -225,7 +228,7 @@ nuclio:
 #      password: somepass
 
 analytics:
-  enabled: true
+  enabled: false
```

Comment:
Analytics is currently enabled by default, please revert the change

Reply:
will do

Reply:
done